<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <author>
    SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia &lt;dev@dragon.best&gt;
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>glaxnimate::gui::StalefilesDialog</class>
 <widget class="QDialog" name="glaxnimate::gui::StalefilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autosave Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="list_widget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_open">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset theme="document-revert">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete_all">
       <property name="text">
        <string>Delete All</string>
       </property>
       <property name="icon">
        <iconset theme="edit-clear"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::StalefilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_open</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::StalefilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_delete</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::StalefilesDialog</receiver>
   <slot>delete_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_delete_all</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::StalefilesDialog</receiver>
   <slot>delete_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_widget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>glaxnimate::gui::StalefilesDialog</receiver>
   <slot>current_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete_selected()</slot>
  <slot>delete_all()</slot>
  <slot>current_changed(int)</slot>
 </slots>
</ui>
